src/deploy.js

```diff
@@ -1,15 +1,85 @@
-import { spawn } from './utils';
+import AWS from 'aws-sdk';
+import s3 from 's3-node-client';
+import dotenv from 'dotenv';
+
+const path = require('path');
+
+// default build directory
+const BUILD = 'build/';
```

Comment:
Two options:
1. Make default `-p` be `./build` and `-p` be described as `path/to/build/folder that will be deployed`.
2. Leave `-p` as is and allow for overriding this with a `-b`.

I think that 1 might be cleaner, but I'm up for discussing this.

Reply:
Good point. Currently I changed it to `-b`. This keeps the syntax from the original `deploy` script which might be appreciated by users of the script. Furthermore, i factored the default value out into the `config.js` file. 

As the changes are easy to make, let me know if you prefer the `-p` variant.